src/solver/conv_hip_implicit_gemm_wrw_v4r4_xdlops.cpp

```diff
@@ -681,6 +690,9 @@ bool PerformanceImplicitGemmWrwV4R4Xdlops::IsReallyValid(const ConvolutionContex
 bool PerformanceImplicitGemmWrwV4R4Xdlops::IsFastToBeUsedForTuning(
     const ConvolutionContext& ctx) const
 {
+
+    if(use_spare_set)
```

Comment:
[Question] So the idea is: If there is none "Fast" perfConfigs exist, let's use all? This seems suspicious, because in this case we can't expect good performance from the Solver, even tuned one. Perhaps it is better to make it non-applicable, thus saving host time?

Reply:
@zjing14 ,I think we still need 2 pass due to tuning time is too long
Isn't it?

Reply:
"Fast" is only relative. It means this tuning parameter is relatively fast compare with other tuning parameters. It doesn't indicate if this solver using this tuning parameter is "fast" or "slow" compare to other solvers. @atamazov

Reply:
Okay then.

Reply:
[Recommendation]
> It means this tuning parameter is relatively fast compare with other tuning parameters. 

Rename the function or insert relevant comment explaining this.